src/git.rs

```diff
@@ -29,9 +30,11 @@ fn delta_operation(delta: &DiffDelta) -> Option<(PathBuf, Operation)> {
     }
 }
 
-pub fn build_churn(repo: &Repository, commits: &[Commit]) -> BTreeMap<PathBuf, usize> {
-    commits
-        .iter()
+pub async fn build_churn<'a>(
+    repo: &'a Repository,
+    commits: Vec<Commit<'a>>,
+) -> BTreeMap<PathBuf, usize> {
+    stream::iter(commits)
```

Comment:
Would `stream::iter(commits.iter())` have worked?

Reply:
What's the benefit of doing that?

Reply:
He had to switch from passing `&commits` to `commits` (reference vs owned) because `stream::iter()` takes ownership. But maybe passing `commits.iter()` would have given it something to own (so we could continue to pass the reference)?

I doubt it but it's worth asking.

Reply:
@mike-burns I was able to make the changes you suggested and this worked.

Separately I'd checked the memory usage with this change, as well as the execution time, and there's not really an improvement that's introduced here ☹️ . Not sure if we want to skip this, or if we think moving to tokio is worth the change architecturally.